documentation/README.md

```diff
@@ -23,14 +23,43 @@ yarn start
 
 This command starts a local, wiki themed development server and opens up a browser window. Most changes are reflected live without having to restart the server.
 
-## Including .md file
+## Features And Tools
```

Comment:
Do we want to keep this section for now or replace it with the style guide from the wiki directly and update that one in the next days?

Reply:
We decided on removing the content once the wiki is ready